### PR TITLE
delete useless function for  dbm package

### PR DESCRIPTION
**Pull request by kadisi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind cleanup
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
delete useless function for  dbm package
```

### CODE REVIEW

File: staging/src/github.com/kubeedge/beehive/pkg/core/module.go [hunk 1]
```diff
@@ -45,18 +45,16 @@ func Register(m Module) {
 	}
 }
 
-//
-func isModuleEnabled(m string) bool {
+// IsModuleEnabled indicates whether m is enable, if enabled return true
```
Comment (fisherxu) [inline staging/src/github.com/kubeedge/beehive/pkg/core/module.go]: whether module is enabled

File: edge/pkg/metamanager/metamanager.go [hunk 2]
```diff
@@ -28,11 +29,21 @@ func newMetaManager() *metaManager {
 
 // Register register metamanager
 func Register() {
-	dbm.RegisterModel(MetaManagerModuleName, new(dao.Meta))
 	metamanagerconfig.InitConfigure()
+	InitDBTable()
 	core.Register(newMetaManager())
 }
 
+//InitDBTable create table
```
Comment (fisherxu) [inline edge/pkg/metamanager/metamanager.go]: add space here

Comment (kadisi) [reply edge/pkg/metamanager/metamanager.go]: done

File: staging/src/github.com/kubeedge/beehive/pkg/core/module.go (hunk 1 above)
Comment (kadisi) [reply staging/src/github.com/kubeedge/beehive/pkg/core/module.go]: done

File: edge/pkg/devicetwin/dtclient/sql.go [hunk 3]
```diff
@@ -17,8 +18,13 @@ const (
 
 //InitDBTable create table
 func InitDBTable() {
-	klog.Info("Begin to register twin model")
-	dbm.RegisterModel("twin", new(Device))
-	dbm.RegisterModel("twin", new(DeviceAttr))
-	dbm.RegisterModel("twin", new(DeviceTwin))
+	klog.Info("Begin to register twin db model")
+
+	if !core.IsModuleEnabled("twin") {
```
[the remark this answers is not in the document]
Comment (kadisi) [reply edge/pkg/devicetwin/dtclient/sql.go]: done

File: edge/pkg/common/dbm/db.go [hunk 4]
```diff
@@ -15,69 +12,27 @@ import (
...
-	orm.RunSyncdb(commonconfig.Get().DBName, false, true)
+	orm.RunSyncdb(dbName, false, true)
 
 	// create orm
 	DBAccess = orm.NewOrm()
-	DBAccess.Using(commonconfig.Get().DBName)
-}
-
-// cleanDBFile removes db file
-func cleanDBFile(fileName string) {
```
[the remark this answers is not in the document]
Comment (kadisi) [reply edge/pkg/common/dbm/db.go]: yes, this function is only usefull for test case

File: edge/pkg/common/dbm/db.go [hunk 5]
```diff
@@ -15,69 +12,27 @@ import (
...
 
-//RegisterModel registers the defined model in the orm if model is enabled
-func RegisterModel(moduleName string, m interface{}) {
-	if isModuleEnabled(moduleName) {
-		orm.RegisterModel(m)
-		klog.Infof("DB meta for module %s has been registered", moduleName)
-	} else {
-		klog.Infof("DB meta for module %s has not been registered because this module is not enabled", moduleName)
-	}
+//InitDBManager initialises the database by syncing the database schema and creating orm
```
[the remark this answers is not in the document]
Comment (kadisi) [reply edge/pkg/common/dbm/db.go]: done

File: edge/pkg/common/dbm/db.go [hunk 6]
```diff
@@ -15,69 +12,27 @@ import (
 //DBAccess is Ormer object interface for all transaction processing and switching database
```
[the remark this answers is not in the document]
Comment (kadisi) [reply edge/pkg/common/dbm/db.go]: done

**Pull request closed without merging**